internal/server/kong/ws/config/version_compatibility_test.go

```diff
@@ -6274,6 +6274,95 @@ func TestVersionCompatibility_ProcessConfigTableUpdates(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "removing whole top-level fields",
+			configTableUpdates: map[string][]ConfigTableUpdates{
+				"< 3.1.0": {
+					{
+						Name:     "keys",
+						Type:     Key,
+						Remove:   true,
+						ChangeID: "T102",
```

Comment:
I don't see any change metadata registered for `T102`?

Reply:
these rules are defined for test only; they don't require user-visible metadata.

Reply:
Derp, completely missed over that. Mind me.